metagraph/src/graph/alignment/dbg_aligner.cpp

```diff
@@ -0,0 +1,25 @@
+#include "dbg_aligner.hpp"
+
+namespace mtg {
+namespace graph {
+namespace align {
+
+IDBGAligner::DBGQueryAlignment IDBGAligner::align(const std::string_view query,
+                                                  bool orientation) const {
```

Comment:
what does `orientation` mean? Does it specify that you pass the forward orientation, or it asks to align with reverse complement?

Try to choose a more descriptive name.

Reply:
I've renamed this to `query_orientation` do indicate that it's stating the orientation of the query (relative to the input)

Reply:
I've again renamed it to `is_reverse_complement`. Hopefully that's clear enough